dev/workflow/build_compute.py

```diff
@@ -166,7 +166,7 @@ def get_host_specs(host: Dict) -> Dict:
     specs.queue = host.info.QUEUE
     specs.partition = partition
     specs.native = native
-    specs.machine = host.machine
+    specs.machine = str(host)
```

Comment:
```suggestion
    specs.machine = host
```
`class Host` has a `__str__` method whose responsibility is to return a string representation of the `host` object.

Reply:
This will assign the `host` object to `specs.machine`, not the string representation.  `str()` calls the `__str__` method.

Reply:
Hmm.  I'll try this out with your suggested change to `__str__`.

Reply:
you are probably correct.  The `__str__` is primarily used for `print` of the object.  My suggestion is probably wrong here.

However, I am positive that `__str__` should return a `f"{whatever}"`

Reply:
Agreed and changed.

Reply:
This worked!